api/envoy/config/listener/v3/listener.proto

```diff
@@ -247,6 +247,10 @@ message Listener {
   // The listener's connection balancer configuration, currently only applicable to TCP listeners.
   // If no configuration is specified, Envoy will not attempt to balance active connections between
   // worker threads.
+  //
+  // Consider the scenario that the listener X set :ref: `original_dst <config_listener_filters_original_dst>`,
+  // and the listeners Y1,Y2 set `bind_to_port <envoy_api_field_config.listener.v3.Listener.bind_to_port>` to
+  // false. The balance config should be cleared in listener X and be enabled in Y1 and Y2.
```

Comment:
Can you re-express? I think the first part makes sense, but I think what you're trying to say after `?` is that the "connection balance config should be applied only to Y1 and Y2"? It's unclear what is meant by "cleared".

Reply:
`connection_balance_config` field needs to be cleared and to disable the balancer behavior. 

s/cleared/disabled in the comment?

Reply:
Updated in commit 799d2e1

Reply:
Are you asking the user to disable/clear? Again, the flow of this paragraph is not very clear. A scenario is proposed out-of-the-blue and then a statement is made. I think we should empathize with the reader and start the top of this comment block and have something that makes sense in context.